client/rest-high-level/src/main/java/org/elasticsearch/client/ml/job/config/AnalysisConfig.java

```diff
@@ -130,14 +121,6 @@ public TimeValue getBucketSpan() {
         return bucketSpan;
     }
 
-    public String getCategorizationFieldName() {
```

Comment:
This is an accessor for a different field - was it removed in error?

Reply:
Definitely in error.